src/main/resources/META-INF/plugin.xml

```diff
@@ -4,16 +4,47 @@
   <name>Biome</name>
   <vendor>biomejs</vendor>
   <description><![CDATA[
-    <a href="https://biomejs.dev">Biome</a> plugin for JetBrains IDEs.
+    <h3>Biome IntelliJ Plugin</h3>
 
-    <h3>Features</h3>
+    <p>The Biome IntelliJ Plugin provides comprehensive support for Biome within IntelliJ IDEA.</p>
 
-    <ul>
+    <b>Features:</b>
+     <ul>
         <li>Linting</li>
         <li>Quick Fix (from mouse-over, <code>⌥⏎</code> or <code>Alt+Enter</code>)</li>
         <li>Reformat your code with <code>⌥⇧ ⌘ L</code> or <code>Ctrl+Alt+L</code></li>
+     </ul>
+
+    <p><a href="https://biomejs.dev/">Biome</a> is a fast formatter for JavaScript, TypeScript, JSX, TSX, JSON, CSS, and GraphQL that scores 97% compatibility with Prettier, saving CI and developer time.</p>
+
+    <p>It is also a performant linter for JavaScript, TypeScript, JSX, CSS, and GraphQL, featuring 281 rules from ESLint, TypeScript ESLint, and other sources.</p>
+
+    <p>Not only can you format and lint your code separately, you can do it all at once with a single command!</p>
+
+    <ul>
+        <li>Fast Built with Rust and an innovative architecture inspired by rust-analyzer.</li>
+        <li>Zero configuration needed to get started. Extensive options available for when you need them.</li>
+        <li>Scalable Designed to handle codebases of any size. Focus on growing product instead of your tools.</li>
+        <li>With tight internal integration we are able to reuse previous work and any improvement to one tool improves them all.</li>
+        <li>Actionable & Informative Avoid obscure error messages, when we tell you something is wrong, we tell you exactly where the problem is and how to fix it.</li>
+        <li>Out of the box support for all the language features you use today. First class support for TypeScript and JSX.</li>
     </ul>
-    ]]></description>
+
+    <p><b>Resources: </b> <a href="https://biomejs.dev/">Biome</a>, <a href="https://github.com/biomejs/biome-intellij">Biome Intellij</a>, <a href="https://github.com/biomejs/biome-intellij/issues">Issue Tracker</a></p>
+  ]]></description>
+
+  <change-notes><![CDATA[
+        <h3>Bug Fixes</h3>
+        <ul>
+            <li>Set plugin version in gradle properties (#78)</li>
+            <li>Upload build artifact (#79)</li>
+            <li>Rename artifact with intellij-biome- instead of Biome- prefix (#81)</li>
+        </ul>
+        <h3>Features</h3>
+        <ul>
+            <li>Migrate to IntelliJ Platform Gradle Plugin (#77)</li>
+        </ul>
+    ]]></change-notes>
```

Comment:
I would exclude the change notes from this PR because they should be kept up to date, which something can't do right now

Reply:
Perfect, deleted.